iati/tests/home_functional_tests.py

```diff
@@ -1 +1,55 @@
 """A module of functional tests for the home page."""
+import pytest
+from tests.base_functional_tests import click_obscured
+
+HOME_PAGE = {
+    'title': 'Home',
+    'heading': 'Test International Aid Transparency Initiative',
+    'excerpt': 'This is an excerpt for the Home page'
+}
+
+
+@pytest.mark.django_db
+class TestHomePage():
+    """A container for tests to check functionality of the Home Page."""
+
+    def navigate_to_edit_home_page(self, admin_browser):
+        """Navigate to the editable section of the CMS for the Home Page."""
+        admin_browser.click_link_by_text('Pages')
+        admin_browser.find_by_xpath('//h3').find_by_text('Home').click()
+        admin_browser.click_link_by_text('Home')
+
+    def publish_changes(self, admin_browser):
```

Comment:
Wouldn't this function be already used elsewhere in the tests?

Reply:
@dalepotter This function is known to be duplicated in different ways currently across tests. Because they're currently context-specific and have different implementations in places it's easier to duplicate for now and resolve it when we refactor tests later to bring these kinds of functions into `conftest.py` under `admin_browser`. So changing this is not within the scope of this PR.

Reply:
In this case I'd be tempted to add a `TODO` comment to this function to say that it needs refactoring out.